### PR TITLE
User guides for Keplr, Leap, Ping Pub and CLI

### PR DESCRIPTION
**Pull request by fadeev**
* Keplr setup and token transfer
* Leap setup and delegating
* CLI setup, accounts, balances, delegating, rewards
* Ping Pub delegating

### CODE REVIEW

File: docs/users/cli/account.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 2
+---
+
```
Comment (btroo) [inline docs/users/cli/account.md:4]: maybe at the top of this sort of doc we could have a component for prereqs -- 'This tutorial requires `zetacored`. To get this set up, check out this doc'. or something like that

**Review by btroo — APPROVED** [review]
This is super helpful! Left one comment to perhaps add context to where one would set up zetacored, which could be in a follow-up. Otherwise, LGTM!

File: docs/users/cli/account.md (hunk 1 above)
Comment (fadeev) [reply docs/users/cli/account.md:4]: https://github.com/zeta-chain/docs/pull/211#:~:text=Add%20a%20note%20about%20installing%20the%20CLI

**Comment by fadeev** [comment]
@andresaiello please, review.

**Review by andresaiello — APPROVED** [review]

**Pull request merged**